SPECS/kernel/config

```diff
@@ -296,7 +296,9 @@ CONFIG_KEXEC_CORE=y
 CONFIG_HAVE_IMA_KEXEC=y
 # CONFIG_KEXEC is not set
 CONFIG_KEXEC_FILE=y
-# CONFIG_KEXEC_SIG is not set
+CONFIG_KEXEC_SIG=y
+# CONFIG_KEXEC_SIG_FORCE is not set
```

Comment:
Only available on x86: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/blob/rolling-lts/mariner-3/6.6.57.1/arch/x86/Kconfig#L2038
Requires that signatures ALWAYS be checked. Currently only checks when a signature is present

(Note missing in https://github.com/microsoft/CBL-Mariner-Linux-Kernel/blob/rolling-lts/mariner-3/6.6.57.1/arch/arm64/Kconfig)

Reply:
Clarification: for this PR we are simply enabling kexec signature validation as an option and not forcing for now